agent/ecscni/plugin.go

```diff
@@ -241,23 +242,25 @@ func (client *cniClient) createENINetworkConfig(cfg *Config) (*libcni.NetworkCon
 		IPV6Address:          cfg.ENIIPV6Address,
 		BlockInstanceMetdata: cfg.BlockInstanceMetdata,
 	}
-
-	return client.constructNetworkConfig(eniConf, ECSENIPluginName)
+	networkConfig, err := client.constructNetworkConfig(eniConf, ECSENIPluginName)
+	return defaultENIName, networkConfig, err
```

Comment:
can you add `if err != nil` check here and make sure that you're wrapping the error as well?